custodia/httpd/authenticators.py

```diff
@@ -131,3 +133,62 @@ def handle(self, request):
         self.audit_svc_access(log.AUDIT_SVC_AUTH_FAIL,
                               request['client_id'], dn)
         return False
+
+
+class OpenIDCTokenAuth(HTTPAuthenticator):
```

Comment:
Please add a doc string with an example how to configure and use this plugin with e.g. Ipsilon.